src/service/grpcnode.ts

```diff
@@ -62,6 +64,24 @@ function printService(printer: Printer, service: RPCDescriptor) {
   printer.printLn(`export const ${serviceName}: I${serviceName};`);
 }
 
+function printServer(printer: Printer, service: RPCDescriptor) {
+  const serverName = `${service.name}Server`;
+  printer.printLn(`export interface I${serverName} extends grpc.UntypedServiceImplementation {`);
+  service.methods
+    .forEach(method => {
+      if (!method.requestStream && !method.responseStream) {
+        printer.printIndentedLn(`${method.nameAsCamelCase}(call: grpc.ServerUnaryCall<${method.requestType}, ${method.responseType}>, callback: grpc.sendUnaryData<${method.responseType}>): void;`);
+      } else if (!method.requestStream) {
+        printer.printIndentedLn(`${method.nameAsCamelCase}(call: grpc.ServerWritableStream<${method.requestType}, ${method.responseType}>): void;`);
+      } else if (!method.responseStream) {
+        printer.printIndentedLn(`${method.nameAsCamelCase}(call: grpc.ServerReadableStream<${method.requestType}, ${method.responseType}>, callback: grpc.sendUnaryData<${method.responseType}>): void;`);
+      } else {
+        printer.printIndentedLn(`${method.nameAsCamelCase}(call: grpc.ServerDuplexStream<${method.requestType}, ${method.responseType}>): void;`);
```

Comment:
For better compatibility of types between `grpc` and `@grpc/grpc-js`, us the following method definitions:

- grpc.handleUnaryCall
- grpc.handleServerStreamingCall
- grpc.handleClientStreamingCall
- grpc.handleBidiStreamingCall